k-distribution/include/builtin/domains.k

```diff
@@ -444,39 +444,40 @@ module INT-SYMBOLIC [symbolic, kore]
   imports  INT-COMMON
   imports ML-SYNTAX
 
-  /**
-   * Definability conditions
-   */
-
-  rule
-    #Ceil(@I1:Int /Int @I2:Int)
-  =>
-    {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2)
-  [anywhere]
-
-  rule
-    #Ceil(@I1:Int %Int @I2:Int)
-  =>
-    {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2)
-  [anywhere]
-
-  rule
-    #Ceil(@I1:Int modInt @I2:Int)
-  =>
-    {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2)
-  [anywhere]
-
-  rule
-    #Ceil(@I1:Int >>Int @I2:Int)
-  =>
-    {(@I2 >=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2)
-  [anywhere]
-
-  rule
-    #Ceil(@I1:Int <<Int @I2:Int)
-  =>
-    {(@I2 >=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2)
-  [anywhere]
+  // Definability Conditions
+  rule #Ceil(@I1:Int /Int   @I2:Int) => {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2) [anywhere]
+  rule #Ceil(@I1:Int %Int   @I2:Int) => {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2) [anywhere]
+  rule #Ceil(@I1:Int modInt @I2:Int) => {(@I2 =/=Int 0) #Equals true} #And #Ceil(@I1) #And #Ceil(@I2) [anywhere]
+  rule #Ceil(@I1:Int >>Int  @I2:Int) => {(@I2 >=Int 0)  #Equals true} #And #Ceil(@I1) #And #Ceil(@I2) [anywhere]
+  rule #Ceil(@I1:Int <<Int  @I2:Int) => {(@I2 >=Int 0)  #Equals true} #And #Ceil(@I1) #And #Ceil(@I2) [anywhere]
```

Comment:
These must have attribute `simplification`. I don't think `anywhere` is necessary: it should have no effect because `#Ceil` isn't a constructor. However, it shouldn't hurt anything.

Reply:
I've incorporated the update #1305 so this shouldn't be an issue anymore.